challenge.js

```diff
@@ -23,7 +23,7 @@
  *	Describe E's relationship to A.
  *
  *	Answer below.
- */
+ */E is the 'grandchild' of A.  A is the parent of D and D is the parent of E. So E is an element of D which is an element of A. E is a second element of A?
```

Comment:
Yep. Grandchild is the preferred term. Second element suggests immediate child. 1